include/nano_gicp/impl/lsq_registration_impl.hpp

```diff
@@ -118,7 +119,7 @@ namespace nano_gicp
             Eigen::Isometry3d delta;
             if(!step_optimize(x0, delta))
             {
-                std::cerr << "lm not converged!!" << std::endl;
+                std::cerr << "lm not converged!!\n" << std::flush;

```

Comment:
Is this actually any more performant?

Reply:
That one was more to keep my static analyzer happy